fuse/readonly/readonly_unix.go

```diff
@@ -181,8 +181,11 @@ func (s *Node) Read(req *fuse.ReadRequest, resp *fuse.ReadResponse, intr fs.Intr
 	if err != nil {
 		return err
 	}
-	n, err := io.ReadFull(r, resp.Data[:req.Size])
+	n, err := r.Read(resp.Data[:req.Size])
```

Comment:
`r.Read(..)` is probably bad, given it may return less than `req.Size` bytes. I like the approach @cryptix suggests.